### PR TITLE
Make it all actually build/test, restricting to python2 for now

### PR DESCRIPTION
**Pull request by yarikoptic**

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,9 +13,8 @@ source:
 
 build:
   number: 0
-  skip: True  # [py>3]
+  skip: True  # [not py2k]
   script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipe/meta.yaml]: Doesn't look like `setuptools` is used to build the package. Please drop it from `requirements/build` and please cut this line down to `python setup.py install`.

**Comment by jakirkham** [comment]
cc @dougalsutherland @synapticarbors

**Comment by jakirkham** [comment]
Also please drop `setuptools` from `requirements/build`.

**Comment by djsutherland** [comment]
Some test failures are related to not having a GUI (`RuntimeError: Invalid DISPLAY variable`: should probably set `MLPBACKEND=agg` or something), but some seem like bugs in `mvpa2`: `TypeError: 'numpy.float64' object cannot be interpreted as an index` and a bunch of `ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`.

**Comment by yarikoptic** [comment]
Thank you @dougalsutherland 
I am on top of it.  Forgot about those numpy issues, will prep/issue a new upstream release and already tuned up locally to not require DISPLAY.  I will buzz you guys whenever things are green ;)

**Comment by yarikoptic** [comment]
A the day has come @dougalsutherland -- I fixed up all the tests so the beast builds/tests all around! whoohoo.  I think it is ready now for the review and merge.  Thank you for  the guidance and patience ;)

**Comment by djsutherland** [comment]
Awesome @yarikoptic, looks good to me.

Since you're the maintainer, you should go ahead and click merge – I actually don't have permissions to do that. :)

**Comment by yarikoptic** [comment]
ha -- I haven't noted this elephant! ;)  Thanks @dougalsutherland

**Pull request merged**